inlang/guides/how-to-setup-inlang-for-your-project/marketplace-manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "guide.niklasbuchfink.howToSetupInlang",
```

Comment:
id are not required for guides. we will give content random ids that are immutable. @floriandwt are you progressing with ids for content/products?

Reply:
@samuelstroschein What do you have in mind for the guide manifests exactly?
The last time we talked, we decided to treat them like items for easier maintainability, and just show them in an extra section and under the route `/g/`. 

If you mean the uniqueIDs, they are already implemented as we have sketched in excalidraw: https://inlang.com/m/2qj2w8pu/app-inlang-cli

Reply:
But I see, there is a bug. Gonna fix it.

Reply:
A perfect, I meant the ids yes! Great to see this implemented.